docs/reference/semaphore-changelog.md

```diff
@@ -8,6 +8,25 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of February 1, 2021
+**(Improved) macOS Xcode 12 image update**  
+**(New)** Xcode 12.4 is installed and set as default version. To switch to any other supported version simply use `xcversion select <version>`.
+**(Updated)** Updated packages:
+ 
+- Fastlane 2.170.0 -> 2.172.0
+- Cocoapods 1.10.0 -> 1.10.1
+
+To learn more about this image, check our [macOS Xcode 12 page](https://docs.semaphoreci.com/ci-cd-environment/macos-xcode-12-image/).
+
+**(Updated)** Docker image:
+New convenience Docker image available in Semaphore's [Semaphore Container Registry](https://docs.semaphoreci.com/ci-cd-environment/semaphore-registry-images/).
```

Comment:
This section has a grammar issue, it's not clear whether there's _a_ new image, or several. Also just say _available in the Semaphore Container Registry_.

Reply:
Noted for the future. I updated this section as a part of this week's changelog update. Could you review it in this PR please: https://github.com/semaphoreci/docs/pull/950 and make sure it looks ok now.